src/App.tsx

```diff
@@ -18,34 +22,22 @@ function App() {
 
       if (!event.target.value) return;
 
-      let skylink = "";
-
       try {
-        skylink = parseSkylink(event.target.value);
+        client
+          .getSkylinkUrl(event.target.value, { subdomain: true })
+          .then((base32Url) => setBase32Skylink(base32Url))
+          .catch((error) => {
+            setBase32Skylink("");
+            setErrorMessage(error.message);
+          });
```

Comment:
use `await` - the `catch` is already there so you don't need to handle it separately